heartbeat/IPsrcaddr

```diff
@@ -177,7 +197,14 @@ srca_start() {
 
 		$CMDCHANGE $ROUTE_WO_SRC src $1 || \
 			errorexit "command '$CMDCHANGE $ROUTE_WO_SRC src $1' failed"
+
 		rc=$?
```

Comment:
You should return early if it fails here.

Reply:
The `errorexit` does that. Custom function that the creator of this RA wrote.

```
errorexit() {
        ocf_exit_reason "$*"
        exit $OCF_ERR_GENERIC
}
```

Reply:
Ah. I didnt think of that :)